src/app/admin/user-management/user-management.styl

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+$title-color = #dfdfdf
```

Comment:
maybe we can reuse existing color definition among
  "$cat-gray-color": "#E4E4E4",
  "$mouse-gray-color": "#D3D3D3",
  "$light-gray-color": "#F1F1F1",